src/components/dictionaryConcepts/containers/EditConcept.jsx

```diff
@@ -334,6 +393,15 @@ Concept
               />
               )
               }
+              <GeneralModel
+                title="Confirm Delete?"
+                content={`Are you sure you want to delete ${mapp}`}
```

Comment:
Is it the concept you are deleting or its mapping?

Reply:
it is the mapping

Reply:
So how does the user know that?

Reply:
The user click remove under the mapping section. So it should be clear

Reply:
That is obvious to you because you are a developer. It is never the same with end users.

Reply:
I have added a more descriptive title

Reply:
The error message is still confusing. If and end user gets a message of are you sure you want to delete MALARIA? You will be shocked to find out that a good number of them will think you are deleting the concept itself. Not all of them look at the title.

Reply:
I have modified the title to indicate that a mapping is been removed with the mapping name provided